crates/cairo-lang-language-server/src/ide/completion/completions.rs

```diff
@@ -282,7 +282,7 @@ fn completion_for_method(
 
     let completion = CompletionItem {
         label: format!("{}()", name),
-        insert_text: Some(format!("{}(", name)),
+        insert_text: Some(format!("{}(", name))),
```

Comment:
Is that what you meant ?
```suggestion
        insert_text: Some(format!("{}()", name)),
```

Reply:
Yes, because the statement gets to fully close.